lib/supavisor/handlers/proxy/client.ex

```diff
@@ -378,7 +379,7 @@ defmodule Supavisor.Handlers.Proxy.Client do
             case Db.send_startup(sock, auth, tenant) do
               :ok ->
                 HandlerHelpers.active_once(sock)
-                auth = Map.put(auth, :ip_ver, ip_ver)
+                auth = Map.put(auth, :ip_version, ip_ver)
```

Comment:
There are :inet or :inet6 in ip_ver, and :ip4 or :ip6 in ip_version

Reply:
That change wasn't meant to be there, so I will clean that up and update.

Reply:
I will put data in defstruct, so it will be easy to refactor